### DISCUSSION TITLE
How to toggle the sidebar when in an input field?

### DISCUSSION THREAD

**Discussion by halloleo** [saulpw/visidata#2505] [Q&A] [answered]
The sidebar pesters sometimes a bit by taking up valuable screen estate. I know in general mode I can switch it off by pressing `b`. But how can I do so when I'm in an input field? 

For example, I press`<space>` to get to the command list. Immidiately the sidebar pops up (althoughh I had toggled iyt off before!) and now it is quite a bit in the way:

<img width="811" alt="Screenshot 2024-08-27 at 6 54 00 PM" src="https://github.com/user-attachments/assets/31dcfe35-88c2-4be3-abeb-a3cd810f6c6f">

How can I now toggle the sidebar off while being in the input field?

**Comment by midichef** [comment, marked as answer]
To get rid of it, press `Ctrl+G`. If you press it repeatedly, it will cycle through different levels of help.

**Reply by cool-RR** [reply]
Works, thank you.
